src/main/java/com/doongji/nestalk/controller/v1/user/UserRestController.java

```diff
@@ -59,4 +62,18 @@
         String email = userService.findEmailByNameAndPhone(request.getName(), request.getPhone());
         return ResponseEntity.ok(new FindEmailResponse(email));
     }
+
+    @ApiOperation(value = "회원 정보 조회")
+    @GetMapping(path = "user/details")
```

Comment:
```suggestion
    @GetMapping(path = "user")
```
참고하실만한 링크를 걸어둘게요
https://gmlwjd9405.github.io/2018/09/21/rest-and-restful.html

Reply:
감사합니다 👍